crates/core/src/live_socket/tests/navigation.rs

```diff
@@ -0,0 +1,316 @@
+use std::sync::{Arc, Mutex};
+
+use crate::live_socket::navigation::*;
+use crate::live_socket::LiveSocket;
+use pretty_assertions::assert_eq;
+use reqwest::Url;
+use serde::{Deserialize, Serialize};
+
+use crate::dom::Document;
+
+macro_rules! assert_doc_eq {
+    ($gold:expr, $test:expr) => {
+        let gold = Document::parse($gold).expect("Gold document failed to parse");
+        let test = Document::parse($test).expect("Test document failed to parse");
+        assert_eq!(gold.to_string(), test.to_string());
+    };
+}
```

Comment:
Wait, can this not be done via `use super::assert_doc_eq`?

Reply:
I should dedup this you are right.